src/pages/Comments/index.tsx

```diff
@@ -0,0 +1,166 @@
+import { Grid } from "@material-ui/core";
+import {
+    AccessTime as AccessTimeIcon,
+    Person as PersonIcon,
+    QuestionAnswer as QuestionAnswerIcon,
+} from "@material-ui/icons";
+import BoxBase from "src/components/Boxs/BoxBase";
+import CommentForm from "src/components/Comments/CommentForm";
+import DividerBase from "src/components/Dividers/DividerBase";
+import FooterBase from "src/components/Footers/FooterBase";
+import CircularProgressBase from "src/components/Progress/CircularProgressBase";
+import TypographyBase from "src/components/Typography/TypographyBase";
+
+import useCommentsList from "./hooks/useCommentsList";
+import useInsertComment from "./hooks/useInsertComment";
+import usePostDetail from "./hooks/usePostDetail";
+
+import CommentList from "src/pages/Comments/CommentsList";
+import LocalStorageUtils from "src/utils/LocalStorageUtils";
+
+const Comments = () => {
+    const user = LocalStorageUtils.getUser();
+    const postId = "727624bd-a537-4d37-8429-b9b8be310e1c";
+
+    const {
+        data: postDetailData,
+        isLoading: isPostDetailLoading,
+        isError: isPostDetailError,
+    } = usePostDetail(postId);
+    const {
+        data: commentsData,
+        isLoading: isCommentsLoading,
+        refetch: refetchComments,
+    } = useCommentsList(postId);
+    const { isLoading, mutate: insertCommentQuery } = useInsertComment(refetchComments);
+

```

Comment:
I think load list of comments from post detail is more easy.
![image](https://user-images.githubusercontent.com/58205909/130461287-8b85acf9-5941-437f-a4b5-3b993bae66a0.png)

Reply:
Oh okay 👍 I keep asking that what is ..._by_pk means :))

Reply:
in my opiniopn, pk = primary key :vv